scipy/stats/tests/test_hypotests.py

```diff
@@ -1019,11 +1019,11 @@ class TestBoschlooExact:
             ([[10, 5], [10, 1]], (0.1652174, 0.08565611)),
             ([[5, 0], [1, 4]], (1, 1)),
             pytest.param(
-                [[0, 1], [3, 2]], (0.5, 0.34375),
-                marks=pytest.mark.xfail_on_32bit(
-                    "This test is sensitive to machine epsilon level",
-                    reason="numerical error"),
-            ),
+                 [[0, 1], [3, 2]], (0.5, 0.34375),
```

Comment:
Why did you increase the indentation (here and below) from 4 to 5 spaces?  Generally 4 is preferred.

Reply:
Ok, it slipped from my eyes when I wrote this. Corrected it in 920bbf3c55f5da59dab237c42ba23e4c112b20d2